extensions/wikia/AdEngine/js/lookup/prebid/prebid.js

```diff
@@ -19,7 +23,9 @@ define('ext.wikia.adEngine.lookup.prebid', [
 	'use strict';
 	var adUnits = [],
 		biddersPerformanceMap = {},
-		prebidLoaded = false;
+		prebidLoaded = false,
+		prebidLazy = adContext.get('opts.isBLBLazyPrebidEnabled') && !uapContext.isFanTakeoverLoaded(),
+		prebidLazyLoaded = false;
```

Comment:
I would consider renaming to something like in comment before (with `is...`), but I'm not sure

Reply:
Done.